staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go

```diff
@@ -69,6 +69,8 @@ type KubeSchedulerConfiguration struct {
 	// then scheduler stops finding further feasible nodes once it finds 150 feasible ones.
 	// When the value is 0, default percentage (5%--50% based on the size of the cluster) of the
 	// nodes will be scored.
+	// Note: It will be overridden by the profile-level score percentage when that is available.
+	// DEPRECATED: This will be removed in v1beta2, please use scheduler profiles to configure it instead.
```

Comment:
The TODO for removal should go here. Remember to put it in it's own comment block.

Reply:
It should be removed. It's marked deprecated in v1beta1